### PR TITLE
Created Project

### PR DESCRIPTION
**Pull request by ipykalo**
Installed Angular CLI;
Installed Material UI;
Configured project to requirements;
Created 'header' component;
Created login component;
Created page-not-found component;
Created service login;
Added validation to login form;

### CODE REVIEW

File: src/app/modules/login-module/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    email: ['', [
+      Validators.required,
+      Validators.minLength(4)
+    ]],
+    password: ['', [
+      Validators.required,
+      Validators.minLength(6)
+    ]]
+  })
+
```
Comment (denis-starchenko) [inline src/app/modules/login-module/login/login.component.ts]: Please move code for form creation to separate private method `createForm` and run his in `ngOninit` method

File: src/app/modules/login-module/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    email: [
+      {type: 'required', message: 'Email is required'},
+      {type: 'minlength', message: 'Email must be at least 4 characters long'},
+    ],
+    password: [
+      {type: 'required', message: 'Password is required'},
+      {type: 'minlength', message: 'Password must be at least 6 characters long'},
+    ]
+  }
+
```
Comment (denis-starchenko) [inline src/app/modules/login-module/login/login.component.ts:36]: Please create file `messages.ts` in folder `constants` on module level and move this code to there.

File: src/app/modules/login-module/login/login.component.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  justify-content: center;
+  align-items: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: #ffffff;
```
Comment (denis-starchenko) [inline src/app/modules/login-module/login/login.component.scss]: Please create file `colors.scss` in folder `scss/constants` on  `src` folder level and create scss variables for colors.

File: src/app/modules/login-module/login/login.component.ts (hunk 1 above)
Comment (ipykalo) [reply src/app/modules/login-module/login/login.component.ts]: done

File: src/app/modules/login-module/login/login.component.ts (hunk 2 above)
Comment (ipykalo) [reply src/app/modules/login-module/login/login.component.ts:36]: done

File: src/app/modules/login-module/login/login.component.scss (hunk 3 above)
Comment (ipykalo) [reply src/app/modules/login-module/login/login.component.scss]: done

**Pull request merged**